pages/common/cradle-sql.md

```diff
@@ -0,0 +1,27 @@
+# cradle sql
+
+> Manage Cradle SQL databases.
+
+- Rebuild the database schema:
+
+`cradle sql build`
+
+- Rebuild the database schema for a specific package:
+
+`cradle sql build {{package}}`
+
+- Truncate the entire database:
```

Comment:
Doe  this just do a cleanup (like the SQLite `VACUUM;` command), or actually remove records?

Reply:
It removes all records, i.e. a ``TRUNCATE TABLE `table_name`;``

Reply:
I see. Perhaps `Empty the entire database` would be clearer?

Reply:
Sounds ok to me 👍